### ISSUE TITLE
Scheme

### ISSUE THREAD

**Issue by J9s3s** [rekomerio/DrakenAudio#1]
Hi. Is there an electrical connection diagram? Thanks

**Comment by rekomerio** [comment]
Sorry but I have not made any. Pins for DAC can be seen here https://github.com/pschatzmann/arduino-audio-tools/wiki/External-DAC and for CAN you can see from here https://github.com/rekomerio/DrakenAudio/blob/bb59f2c5e6c0e9341ba4239c1d3f4f87cc584218/src/main.cpp#L21C1-L21C106

**Comment by J9s3s** [comment]
Hi. Do you have a DRV134 powered by 12V or 5V?

**Comment by rekomerio** [comment]
5V and do note you will need negative voltage converter such as MC34063A to get -5V

**Comment by J9s3s** [comment]
Thanks a lot for the reply. 
How did you connect the DRV134 to the connector in the trunk? L+(DRV134) to L+(connector), R- to R- and so on?

**Comment by rekomerio** [comment]
Yes exactly, here is the pinout from bluesaab project you can use as reference
![image](https://github.com/user-attachments/assets/cd8e70b3-6b0f-42b1-ad12-9f763858ddea)

**Comment by J9s3s** [comment]
Hello. Do I need to configure DRV134 somehow? I have strong wheezing when the music is on. And when turned off, you can hear a squeak.

**Comment by rekomerio** [comment]
I never had that type of issue. Could it be a grounding issue?

**Comment by J9s3s** [comment]
Have you connected the GND on the output pins of the left and right channels to the ground?
